http/method.ts

```diff
@@ -0,0 +1,349 @@
+// Copyright 2018-2024 the Deno authors. All rights reserved. MIT license.
+// This module is browser compatible.
+// This module is generated from {@link https://www.iana.org/assignments/http-methods/http-methods.xhtml#methods | IANA Hypertext Transfer Protocol (HTTP) Method Registry}
+
+/**
+ * HTTP Methods as defined by RFC 9110
+ *
+ * @see {@link https://www.iana.org/go/rfc9110 | RFC9110, Section 9.3}
+ */
+export const HttpMethodRfc9110 = {
```

Comment:
Please move all entries to this object. Users can differentiate where the definitions come from from JSDocs.
```suggestion
export const METHODS = {
```